qt/scientific_interfaces/ISISReflectometry/GUI/Preview/PreviewModel.h

```diff
@@ -52,9 +52,6 @@ class MANTIDQT_ISISREFLECTOMETRY_DLL PreviewModel final : public IPreviewModel {
   void exportReducedWsToAds() const override;
 
 private:
-  // This should be an optional instead of a point, but we have issues reassigning it because boost::optional doesn't
-  // play well with non-copyables. This should be fixable when we can use std::make_optional, but that is disabled on
-  // Mac right now.
```

Comment:
Tested the GUI and everything is working as expected. The code changes look good. 
I have a question here, is it possible to replace this pointer with std::optional instead of removing the comment? Looks like there is a good reason this comment was left

Reply:
@MohamedAlmaki Yes, that is indeed possible. I've made the change and it worked well in tests.